CactbotOverlay/CactbotOverlay.csproj

```diff
@@ -61,6 +61,14 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>ThirdParty\OverlayPlugin\libs\HtmlRenderer.dll</HintPath>
     </Reference>
+    <Reference Include="Machina, Version=2.1.1.2, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>ThirdParty\Machina.FFXIV\Machina.dll</HintPath>
+    </Reference>
+    <Reference Include="Machina.FFXIV, Version=2.1.3.10, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>ThirdParty\Machina.FFXIV\Machina.FFXIV.dll</HintPath>
+    </Reference>
```

Comment:
Ravahn never published Machina builds (as far as I know) so everyone would have to make their own Machina builds or unpack them from the FFXIV plugin. Both sound like a bad solution.
In OverlayPlugin I used reflection to work around this.